File: Python Code/Daily_Flash_Code_0/Week 2/24Jan/10-DailyFlash_Solutions/10-DailyFlash_Solutions/24_Jan_Solutions_Three/Java/prog3.java
import java.util.Scanner;

class Demo {
	public static void main(String[] args) {
		System.out.println("Enter num : ");
		Scanner sc = new Scanner(System.in);
		int num = sc.nextInt();
		for(int i = 1; i <= num; i++) {
			System.out.println(i % 2 == 1 ? "Cube of " + i + " : " + i * i * i :"");
		}
	}
}

File: Python Code/Daily_Flash_Code_0/Week 2/24Jan/10-DailyFlash_Solutions/10-DailyFlash_Solutions/24_Jan_Solutions_Three/C/prog3.c
#include<stdio.h>

int main(void) {
	int num;
	printf("Enter num : ");
	scanf("%d", &num);	
	for(int i = 1; i <= num ; i++) {
		if(i % 2 == 1)
			printf("Cube of %d : %d\n", i, i * i * i);
	}
	return 0;
}
